packages/dd-trace/test/appsec/iast/utils.js

```diff
@@ -21,6 +21,13 @@ function testInRequest (app, tests) {
   let appListener
   const config = {}
 
+  before(() => {
+    return agent.load('http', undefined, { flushInterval: 1 })
```

Comment:
loading and closing the agent only once instead of for each test, reduces the flakyness.